### ISSUE TITLE
Make sure the quota work as expected after introducing the OCI work

### ISSUE THREAD

**Issue by ywk253100** [goharbor/harbor#10065]
- [x] Make sure it can work for index/manifest list
- [x] ~~Remove sync API~~ Refactor sync API to refresh mode
- [x] Should the count quota apply to artifact rather than tag(especially if we support manage the untagged artifacts)?

**Comment by xaleeks** [comment]
I think it should be on the tag because we already control actual disk consumption through the disk quota parameter, so controlling it on tag number gives it an additional dimension.

**Comment by ywk253100** [comment]
Got error when pushing manifest list:
```
2019-12-31T16:05:25+08:00 [ERROR] [/replication/transfer/image/transfer.go:340]: failed to push manifest of image test/hello-world:latest: http error: code 500, message {"errors":[{"code":"InternalError","message":"Error occurred when to handle request in size quota handler: unsupported content type for manifest: application/vnd.docker.distribution.manifest.list.v2+json","detail":"Error occurred when to handle request in size quota handler: unsupported content type for manifest: application/vnd.docker.distribution.manifest.list.v2+json"}]}
```

**Comment by renmaosheng** [comment]
There is a notification needs to be fired which is tracked in issue#10193. Need to perform a thorough testing before close this.

**Comment by heww** [comment]
Closed as notifaction is not supported by #11123
